### DISCUSSION TITLE
"message": { "tag":  null } - what it does

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#2388] [Help with using Postal]
the api docs mention: 

> tag
> The tag of the e-mail

that is a bit vague, I would say.
```
    "message": {
      "id": 1686,
      "timestamp": 1679396413.355525,
      "spam_status": "NotSpam",
      "tag": null }
```
Can someone explain me the TAG? How can it be used and or whats the purpose of it? This is from a webhook message.

**Comment by willpower232** [comment]
You can tag messages when sending them with Postal to help you manage messages. I can't say I've used the feature personally though.
